Add support for aarch64, powerpc64 and riscv64.
Use OpenBSD-specific files for arm (TBD).

Index: gdb/configure.host
--- gdb/configure.host.orig
+++ gdb/configure.host
@@ -81,6 +81,7 @@ case "${host}" in
 
 aarch64*-*-linux*)	gdb_host=linux ;;
 aarch64*-*-freebsd*)	gdb_host=fbsd ;;
+aarch64*-*-openbsd*)	gdb_host=obsd64 ;;
 
 alpha*-*-linux*)	gdb_host=alpha-linux ;;
 alpha*-*-netbsdaout* | alpha*-*-knetbsdaout*-gnu)
@@ -93,7 +94,7 @@ arm*-*-freebsd*)	gdb_host=fbsd ;;
 arm*-*-linux*)		gdb_host=linux ;;
 arm*-*-netbsd* | arm*-*-knetbsd*-gnu)
 			gdb_host=nbsdelf ;;
-arm*-*-openbsd*)	gdb_host=nbsdelf ;;
+arm*-*-openbsd*)	gdb_host=obsd ;;
 
 hppa*-*-linux*)		gdb_host=linux ;;
 hppa*-*-netbsd*)	gdb_host=nbsd ;;
@@ -140,12 +141,14 @@ powerpc*-*-freebsd*)	gdb_host=fbsd ;;
 powerpc-*-netbsdaout* | powerpc-*-knetbsd*-gnu)
 			gdb_host=nbsd ;;
 powerpc-*-openbsd*)	gdb_host=obsd ;;
+powerpc64-*-openbsd*)	gdb_host=obsd64 ;;
 
 powerpc64*-*-linux*)	gdb_host=ppc64-linux ;;
 powerpc*-*-linux*)	gdb_host=linux ;;
 
 riscv*-*-freebsd*)	gdb_host=fbsd ;;
 riscv*-*-linux*)	gdb_host=linux ;;
+riscv64-*-openbsd*)	gdb_host=obsd64 ;;
 
 s390*-*-linux*)		gdb_host=linux ;;
 
